taskcluster/app_services_taskgraph/transforms/module_build.py

```diff
@@ -0,0 +1,44 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from __future__ import absolute_import, print_function, unicode_literals
+
+from taskgraph.transforms.base import TransformSequence
+
+transforms = TransformSequence()
+
+@transforms.add
+def release_upload_symbols(config, tasks):
+    for task in tasks:
+        if config.params["tasks_for"] == u"github-release":
```

Comment:
Nit: This PR doesn't take `github-release`s into account. Let's remove this logic from the PR. I don't think this is the right way to go. In my opinion, we should have 2 `release-type`s: a regular one and a release one. The modifications done below can be bubbled up to `kind.yml`. See android-components, for example: https://github.com/mozilla-mobile/android-components/blob/7e38ed297b76e9cdc42b88e2af5a2cdad20af146/taskcluster/ci/build/kind.yml#L21-26